deployment/ccip/changeset/v1_6/config.go

```diff
@@ -17,92 +17,40 @@ import (
-			DeltaStage:                              60 * time.Second, // TransmissionDelayMultiplier overrides DeltaStage
-			Rmax:                                    globals.OcrRMax,
-			MaxDurationQuery:                        globals.OcrMaxDurationQuery,
-			MaxDurationObservation:                  35 * time.Second,
-			MaxDurationShouldAcceptAttestedReport:   globals.OcrMaxDurationShouldAcceptAttestedReport,
-			MaxDurationShouldTransmitAcceptedReport: 10 * time.Second,
-		},
+		OCRParameters:        globals.CommitOCRParamsForEthereum,
 		CommitOffChainConfig: &globals.DefaultCommitOffChainCfg,
 	}
 
+	DefaultOCRParamsForExecForNonETH = CCIPOCRParams{
+		OCRParameters:         globals.ExecOCRParams,
+		ExecuteOffChainConfig: &globals.DefaultExecuteOffChainCfg,
+	}
+
 	DefaultOCRParamsForExecForETH = CCIPOCRParams{
-		OCRParameters: types.OCRParameters{
-			DeltaProgress:                           100 * time.Second,
-			DeltaResend:                             globals.OcrDeltaResend,
-			DeltaInitial:                            globals.OcrDeltaInitial,
-			DeltaRound:                              90 * time.Second,
-			DeltaGrace:                              globals.OcrDeltaGrace,
-			DeltaCertifiedCommitRequest:             globals.OcrDeltaCertifiedCommitRequest,
-			DeltaStage:                              60 * time.Second, // TransmissionDelayMultiplier overrides DeltaStage
-			Rmax:                                    globals.OcrRMax,
-			MaxDurationQuery:                        globals.OcrMaxDurationQuery,
-			MaxDurationObservation:                  20 * time.Second,
-			MaxDurationShouldAcceptAttestedReport:   globals.OcrMaxDurationShouldAcceptAttestedReport,
-			MaxDurationShouldTransmitAcceptedReport: 8 * time.Second,
-		},
+		OCRParameters:         globals.ExecOCRParamsForEthereum,
 		ExecuteOffChainConfig: &globals.DefaultExecuteOffChainCfg,
 	}
 
 	// Used for only testing with simulated chains
 	OcrParamsForTest = CCIPOCRParams{
 		OCRParameters: types.OCRParameters{
-			DeltaProgress:                           globals.DeltaProgress,
-			DeltaResend:                             globals.DeltaResend,
-			DeltaInitial:                            globals.DeltaInitial,
-			DeltaRound:                              globals.DeltaRound,
-			DeltaGrace:                              globals.DeltaGrace,
-			DeltaCertifiedCommitRequest:             globals.DeltaCertifiedCommitRequest,
-			DeltaStage:                              globals.DeltaStage,
-			Rmax:                                    globals.Rmax,
-			MaxDurationQuery:                        globals.MaxDurationQuery,
-			MaxDurationObservation:                  globals.MaxDurationObservation,
-			MaxDurationShouldAcceptAttestedReport:   globals.MaxDurationShouldAcceptAttestedReport,
-			MaxDurationShouldTransmitAcceptedReport: globals.MaxDurationShouldTransmitAcceptedReport,
+			DeltaProgress:                           10 * time.Second,
```

Comment:
should it be moved to ocr3.go?

Reply:
Replied on slack. I will try to get rid of this entirely in a followup